src/components/RenderBooks.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable camelcase */
+import PropTypes from 'prop-types';
```

Comment:
- [ ] Hi, I would advise you not to disable Linter because Linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Also, the general requirements prohibit disabling it.
- Note: Check if your app is working accordingly(Adding and Removing books ) before requesting for re-review, this will help you to save time.